react/components/EditorContainer/Sidebar/ConfigurationList/ContentEditor/ConditionControls/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+export type DateRange = {
```

Comment:
In this case, what is actually the difference between defining this as `type` instead of `interface`? We're usually using `interface`, so I'd stick with it, but I don't really know the difference.

Reply:
Our linter usually complains when we use types instead of interfaces. If I'm not mistaken, types can be used for pretty much anything, while interfaces can only define objects. For me, the main difference is that interfaces can be implemented, so my decisions are solely based on whether or not whatever I'm declaring should allow that. Since, at least for now, `DateRange` won't need to be implemented, it's just a type. Nothing is set in stone though, so eventually changing it to an interface is perfectly acceptable if needed.